src/groups/mqb/mqbnet/mqbnet_tcpsessionfactory.cpp

```diff
@@ -502,6 +502,8 @@ void TCPSessionFactory::negotiationComplete(
                              statusCode,
                              d_allocator_p);
         channel->close(status);
+
+        logOpenSessionTime("", channel);
```

Comment:
This is a case when negotiation is failed and session incomplete, all available information will be logged in the lines above. Instead of session we can print channel info in this case, but not sure if it is valuable.

Reply:
Considered to add channel info in this case for better log readability.